handbook/direction/code-graph/code-insights/index.md

```diff
@@ -1,44 +1,220 @@
-# Code insights goals and roadmap
+# Code Insights Direction
 
-The Code insights team focuses on bringing code insights to life as a valuable new Sourcegraph product.
+Code Insights is an upcoming product that lets you track and understand what's in your code and how it changes over time.
 
-The team's goals and roadmap contain the features and customer signals that guide our work.
+This page outlines the vision, strategy, and goals of the Code Insights team.
 
-## Goals for FY22
+Quick links:
 
-See also our [completed goals](../../../engineering/code-graph/code-insights/goals_completed.md).
+- [Code Insights Backlog](https://github.com/orgs/sourcegraph/projects/200/views/13)
+- Latest demo – **TODO check with marketing if we're allowed to share a demo here**
```

Comment:
Amie said she prefers not to include a product demo even of we are making it clear we're currently in beta (and we are!).

Reply:
Got it, will modify, thanks!

Reply:
```suggestion
- Latest demo – private since the product is in beta; slack the  #code-insights team or email feedback@sourcegraph.com if you're curious! 
```